### PR TITLE
Updates and additions to community distributions page

### PR DESCRIPTION
**Pull request by bennyvasquez**
## Description

Added Gioardi
Updated the information around Cinc.
Added a call to open pull request for updates.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: distributions/distribution-list.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Distributions
 
-Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have also created alternative distributions of Chef's Open Source products. These are listed below.
+Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have created alternative distributions of Chef's Open Source products. These are listed below.
 
-## cc-build
+## Cinc
 
-This includes community distributions of Chef's software (including Chef Infra, InSpec, Habitat, and Automate).
+The Cinc project currently includes community distributions of Chef Infra-client (Cinc Client) and Chef InSpec (Cinc Auditor).
```
Comment (ramereth) [inline distributions/distribution-list.md]: Could you please add our website as well? https://cinc.sh

Comment (bennyvasquez) [reply distributions/distribution-list.md]: done!

File: distributions/distribution-list.md [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
 * [Source Code](https://github.com/biome-sh/biome)
+* [Project Site](https://biome.sh/)
+
+## Goiardi
+
+Not technically a community distribution, this Goiardi is a feature-limited implemention of Chef Server, written in [Go](https://golang.org/).
+
+* [Source Code](https://github.com/ctdk/goiardi)
+* [Project Site](http://goiardi.gl/)
+
```
Comment (tas50) [inline distributions/distribution-list.md]:
```suggestion
```

File: distributions/distribution-list.md [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
 ## Biome
 
 This is a community distribution of Habitat.
 
 * [Source Code](https://github.com/biome-sh/biome)
+* [Project Site](https://biome.sh/)
+
+## Goiardi
+
+Not technically a community distribution, this Goiardi is a feature-limited implemention of Chef Server, written in [Go](https://golang.org/).
```
Comment (tas50) [inline distributions/distribution-list.md]:
```suggestion
Not technically a community distribution, Goiardi is a feature-limited implementation of Chef Server, written in [Go](https://golang.org/).
```

File: distributions/distribution-list.md [hunk 4]
```diff
@@ -1,16 +1,29 @@
 # Distributions
 
-Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have also created alternative distributions of Chef's Open Source products. These are listed below.
+Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have created alternative distributions of Chef's Open Source products. These are listed below.
 
-## cc-build
+## Cinc
 
-This includes community distributions of Chef's software (including Chef Infra, InSpec, Habitat, and Automate).
+The Cinc project currently includes community distributions of Chef Infra-client (Cinc Client) and Chef InSpec (Cinc Auditor).
```
Comment (tas50) [inline distributions/distribution-list.md]:
```suggestion
The Cinc project currently includes community distributions of Chef Infra Client (Cinc Client) and Chef InSpec (Cinc Auditor).
```

File: distributions/distribution-list.md [hunk 5]
```diff
@@ -1,16 +1,28 @@
...
+* [Project Site](https://biome.sh/)
+
+## Goiardi
+
+Not technically a community distribution, Goiardi is a feature-limited implementation of Chef Server, written in [Go](https://golang.org/).
+
+* [Source Code](https://github.com/ctdk/goiardi)
+* [Project Site](http://goiardi.gl/)
+
+If you know of anything wrong or missing, please feel free to email community@chef.io or open a pull request to update this page!
```
Comment (robbkidd) [inline distributions/distribution-list.md]:
```suggestion
If you have any updates or corrections, please feel free to email community@chef.io or open a pull request to update this page!
```

How about a rewording of what changes people might have?

File: distributions/distribution-list.md [hunk 6]
```diff
@@ -1,16 +1,28 @@
 # Distributions
 
-Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have also created alternative distributions of Chef's Open Source products. These are listed below.
+Along with the [offical Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have created alternative distributions of Chef's Open Source products. These are listed below.
```
Comment (phiggins) [inline distributions/distribution-list.md]:
```suggestion
Along with the [official Chef distributions](https://downloads.chef.io/) of our Open Source projects, members of our community have created alternative distributions of Chef's Open Source products. These are listed below.
```

While you're in here 👼

**Pull request merged**